app/graphql/types/invoices/object.rb

```diff
@@ -52,6 +52,7 @@ class Object < Types::BaseObject
       field :invoice_subscriptions, [Types::InvoiceSubscription::Object]
       field :subscriptions, [Types::Subscriptions::Object]
 
+      field :integration_external_id, String, null: true
```

Comment:
Can we do a small change so that it matches how the other external fields are named?

```suggestion
      field :external_integration_id, String, null: true
```